Model_scripts/Functions.jl

```diff
@@ -1,31 +1,38 @@
 # Defines the functions used to process the model parameters
-"""
-Differential equations formula for finding densities of each pop at next step
-:param indiv: vector of number of individuals of each group
-    (first idx is uninfected hosts; 2:end to each strain)
-:param range: window for each time step
-:param parameters: model parameters
-:returns: differential eqn to solve
-"""
+
+# Statistical functions ######################################################
 function comp_densities(indiv, range, parameters)
+    """
```

Comment:
@kiristern i may be wrong but I believe you were the one who took these code blocks outside of the functions. they should remain inside, as to inform what the function does :)

Reply:
i was following the Documenter.jl format and it says to put the code blocks before the function. i'm okay with moving them back inside

Reply:
they should definitely be outside!

Reply:
yup i see now! my bad, i got too used to the pythonic way of doing things. you guys are right, thanks for clarifying!